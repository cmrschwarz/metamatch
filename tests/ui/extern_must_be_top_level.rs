use metamatch::eval;

fn main() {
    eval! {
        {
            extern let FOO = [1, 2, 3];
        };
        INT_VARIANTS
    };
}
