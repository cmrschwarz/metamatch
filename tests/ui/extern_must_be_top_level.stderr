error: `extern` can only occur at the top level
 --> tests/ui/extern_must_be_top_level.rs:6:13
  |
6 |             extern let FOO = [1, 2, 3];
  |             ^^^^^^
